src/app/app.component.scss

```diff
@@ -28,6 +28,11 @@
   background-image: url("./assets/food.png");
 }
 
+.wall-cell {
+  background-color: var(--light-red);
```

Comment:
I second the border-radius removal.

Reply:
I've merged it and changed the color😄 As for the design - I was thinking about asking Rali about it, because either way they look not so aesthetically pleasing haha

Reply:
Hmmm we could do a purple somewhat thick border and the inside being black/gray?